hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -221,6 +238,21 @@ public boolean isLockAcquisitionFailed() {
    */
   public final class DeleteCmdWorker implements Runnable {
 
+    private long interval;
+
+    public DeleteCmdWorker() {
+      this(HddsConfigKeys.HDDS_COMMAND_DELETE_WORKER_INTERVAL_DEFAULT);
+    }
```

Comment:
I don't think this is needed.

Reply:
Thanks for your comment and review, @adoroszlai .
I will update soon.